src/components/map-marker/map-marker.tsx

```diff
@@ -85,6 +94,7 @@ export class MapMarker implements GxComponent {
   }
 
   componentDidUpdate() {
+    const halfIconWidth = this.iconWidth / 2;
```

Comment:
This code is very similar to the code in the `componentDidLoad` method. 
Can we refactor it a little bit to avoid repetition?

Reply:
That is an upgrade to be done in ```add/MarkerPopUp``` branch. Stay sharp for the #97 PR